braidpool_spec.md

```diff
@@ -146,14 +229,38 @@ primarily to discourage pool hopping. We don't feel that this is needed in the
 modern day and a smoothing function applied to payouts interferes with the
 notion of using shares as a hashrate derivative instrument.
 
-A purely work-weighted proportional algorithm would work for a pure-DAG
+[kp]
+
+I like PPLNS as it does exactly that - prevents pool hopping. By not addressing
+this we are leaving braidpool vulnerable to future innovations in markets and
+pool operations. Why do you feel this is not a problem anymore. If we want to
+leave PPLNS out of braidpool, we need a more convincing argument. I'd argue for
+PPLNS and reference Meni's work.
+
+You could say, braidpool allows for any accounting scheme on top of the
+eventually consistent view of shares. On that note, we should talk about the
+eventually consistency of shares view in the broadcast section. This again makes
+me think we need that overview section with a system architecture diagram where
+each component has clearly defined responsibilities.
+
+[/kp]
+
+A purely work-weighted proportional algorithm will work for a pure-DAG
 blockchain, however we have the problem that some of the beads are blocks in a
 parent blockchain, and the parent blockchain has the property that some blocks
 can be orphans and receive no reward. We must dis-incentivize the creation of
 blocks which might become orphans. One component of this solution is the
 [difficulty retarget algorithm](#difficulty-retarget-algorithm) which maximizes
 throughput while minimizing the number of simultaneous beads.
 
+[kp]
+
+Is this really a problem? We make payouts after 100 blocks, as does bitcoin,
+that is enough time to be sure the block is not going to be orphaned.
+
+
+[/kp]
```

Comment:
Because the DAG keeps all beads, we can receive a bead that is 100 blocks old, and it will be in the DAG and we have to figure out what to do with it.

We also have to punish latency to the extent that they create orphans. I have a formula somewhere of the probability of becoming an orphan as a function of latency. We could use this to decide rewards.